.github/ISSUE_TEMPLATE/issue_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Issue report
+about: Create a report to help us improve MetaMorpheus
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describing the issue**
+What issue did you encounter?
+
+**Reproducing the error**
+Which protein database did you use? What type of mass spectrometry data files did you use? If possible, please provide examples of the datafiles that reproduce the error. 
+
```

Comment:
We can provide a folder that you can use to upload the data. Let us know if the data needs to remain private.

Reply:
Noted!